src/griddyn/solvers/arkodeInterface.cpp

```diff
@@ -324,7 +324,7 @@ void arkodeInterface::initialize (coreTime time0)
 
     // dynInitializeB CVode - Sundials
 
-    int retval = ARKodeSetUserData (solverMem, reinterpret_cast<void *> (this));
+    int retval = ARKodeSetUserData (solverMem, static_cast<void *> (this));
```

Comment:
I _think_ these only need to be static and not reinterpret

Reply:
in reading a little more into this static_cast can cast to void *.  in a few cases things need to be cast to char * or const char *.  this would require the reinterpret_cast<char *>.